### ISSUE TITLE
[Feature Request] Make chips (especially labels) disabled by default

### ISSUE THREAD

**Issue by mitar** [vuetifyjs/vuetify#3817]
### Problem to solve
We used `<v-chip label>` in our app to show labels and only after some user testing we realized that users are complaining that those buttons seem to not do anything. Surprised, we realized that one can click on labels and they get highlighted, but do not do anything.

### Proposed solution
I think chips (especially labels) should be simply disabled by default and if somebody wants to make them selectable or actionable they should enable them. Otherwise this is a very surprising UX where it is unclear that you can have an action over otherwise something which looks static. Labels look very much like buttons which adds additional confusion.

**Comment by FINDarkside** [comment]
Not a good idea, currently it works as expected. You should look into material design: https://material.io/guidelines/components/chips.html

It sounds more like you're using chip for wrong purpose.

**Comment by mitar** [comment]
I do not see "label" type of chip on the link you linked? Where is specified that label should be clickable and it should toggle shadow when clicked, but no other action should be done?

**Comment by MarkFreedman** [comment]
Not so sure... the MD specs mention for behavior: "Selecting a chip opens a full detail view." This isn't specific for "contact chips." MD specs don't show "label" chips as an option, so defaulting to disabled may be fine for this Vuetify-specific chip component enhancement.

**Comment by blalan05** [comment]
This is available in the 2.0 alpha.

**Comment by mitar** [comment]
Maybe I am nitpicking here, but current UX is still confusing a bit here, so I opened a new issue: https://github.com/vuetifyjs/vuetify/issues/10115
